debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorTask.java

```diff
@@ -51,12 +51,13 @@ public ChangeEventSourceCoordinator start(Configuration config) {
 
         Configuration jdbcConfig = connectorConfig.jdbcConfig();
         jdbcConnection = new OracleConnection(jdbcConfig, () -> getClass().getClassLoader());
-        OracleValueConverters valueConverters = new OracleValueConverters(connectorConfig, jdbcConnection);
+        connectorConfig.getAdapter().configure(connectorConfig, jdbcConnection);
```

Comment:
See above: can we push this to instantion time of the adapters?

Reply:
In order to pass the connection, we would need to perform the instantiation lazily.  I'll add a commit doing that and we can decide if we prefer that approach or not.